### PR TITLE
fork

### PR DESCRIPTION
**Pull request by chaicoder0101**

### CODE REVIEW

**Comment by chaicoder0101** [comment]
pls check the clean up

**Review by gyurmey — APPROVED** [review]

**Review by gyurmey — COMMENTED** [review]
okay

**Comment by gyurmey** [comment]
great

File: src/components/pages/ContactMe.js [hunk 1]
```diff
@@ -13,13 +13,6 @@ export default class MyForm extends React.Component {
 		const { status } = this.state;
 		return (
 			<Fragment>
-				{/* <div className='social-links'>
-					<h4 style={{ color: 'black' }}>E-Mail : gyurmey1835@gmail.com</h4>
-
-					<a href='https://github.com/gyurmey' rel='noopener noreferrer' target='_blank'>
```
Comment (gyurmey) [inline src/components/pages/ContactMe.js:19]: good

Comment (chaicoder0101) [reply src/components/pages/ContactMe.js:19]: d

**Review by chaicoder0101 — COMMENTED** [review]
s

**Pull request merged**